.gitmodules

```diff
@@ -3,4 +3,10 @@
 	url = https://github.com/bendavid/narf.git
 [submodule "wremnants-data"]
 	path = wremnants-data
-	url = https://gitlab.cern.ch/cms-wmass/wremnants-data.git
\ No newline at end of file
+	url = https://gitlab.cern.ch/cms-wmass/wremnants-data.git
+[submodule "combinetf2"]
+	path = combinetf2
+	url = git@github.com:wmass/combinetf2.git
```

Comment:
Better to use the https url here to avoid needing to type passwords for ssh keys etc